HIAPI/Models/Attendee.swift

```diff
@@ -21,7 +21,7 @@ public struct Attendee: Codable, APIReturnable {
     public let id: String
     public let firstName: String?
     public let lastName: String?
-    public let email: String
+    public let email: String? // Email should be an optional field because not every GH account has their email set to public
```

Comment:
Technically this email should be required since `Attendee`s provide these at Registration